force-app/main/default/lwc/edaSettings/edaSettings.html

```diff
@@ -2,13 +2,13 @@
     <template if:true={currentUserHasAccess}>
         <div class="slds-grid slds-theme_default eda-height_full">
             <div
-                class="slds-col slds-p-top_x-small slds-no-flex slds-size_4-of-12 slds-scrollable_y eda-height_full"
+                class="slds-col slds-no-flex slds-size_4-of-12 slds-scrollable_y slds-p-top_x-small"
                 data-qa-locator="edaSettingsNavigation"
             >
                 <c-eda-settings-navigation onsettingsnavigation={handleSettingsNavigation}></c-eda-settings-navigation>
             </div>
             <div
-                class="slds-col slds-no-flex slds-size_8-of-12 slds-scrollable_y eda-height_full"
+                class="slds-col slds-no-flex slds-size_8-of-12 slds-scrollable_y slds-p-top_large slds-p-left_small"
```

Comment:
adding paddings here and also move it to the end.

Reply:
Adding the padding to start with here is a good idea. This should keep consistency throughout.